apps/payments/next/project.json

```diff
@@ -57,14 +57,7 @@
       "executor": "@nx/jest:jest",
       "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
       "options": {
-        "jestConfig": "apps/payments/next/jest.config.ts",
-        "passWithNoTests": true
-      },
-      "configurations": {
-        "ci": {
-          "ci": true,
-          "codeCoverage": true
-        }
+        "jestConfig": "apps/payments/next/jest.config.ts"
```

Comment:
Ah, I was going to ask what was up with removing the CI coverage stuff and these Jest bits, but looks like it might have moved into an Nx config file:
https://github.com/mozilla/fxa/blob/823268fb3316041fff4b580b2805a52ded7310ea/nx.json#L129-L137